app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardViewModel.kt

```diff
@@ -139,12 +141,11 @@ class DashboardViewModel(
         exposureNotificationManager.refreshPrerequisitesErrorStatement(ignoreErrors)
     }
 
-    fun unseenChangelogForVersionAvailable(version: String): Boolean {
-        return changelogManager.unseenChangelogForVersionAvailable(version)
-    }
-
-    fun observeExposureSDKReadyToStart(): Observable<Boolean> {
-        return changelogManager.observeExposureSDKReadyToStart().distinctUntilChanged()
+    fun observeCurrentChangelogSeen(): Observable<Boolean> {
```

Comment:
I suggest to call it `observeCurrentChangelogPending` and to move it to the `changelogManager`.

Currently we have `changelogManager.observeIsChangelogSeen()` and `observeCurrentChangelogSeen()` which have almost the same name but different semantics for the case of no current changelog present.

Reply:
I don' agree with moving this part to the `changelogManager` because I need here a lifecycle of dashboard viewModel.

1. Once changelog should be displayed `dashboardViewModel.shouldDisplayWhatsNew = true`
2. Changelog is displayed.
3. Changelog is closed.
4. changelogManager sends seen event.
5. In dashboardViewModel if seen is true AND shouldDisplayWhatsNew is true, then pass this information to check if the framework should be started for the first time.
6. In next start. dashboardViewModel receives seen state as true, but shouldDisplayWhatsNew is false, meaning no checking auto start of the framework.

Reply:
I don't understand the lifecycle argument. I just suggest moving `observeCurrentChangelogSeen` to the `changelogManager`. This is just the stream builder.  You would still (un-)subscribe from the view model. 

Anyaway. Your choice. But please address the semantic/naming issue